pages/integrations/index.mdx

```diff
@@ -0,0 +1,65 @@
+---
+meta:
+  title: Integrations | Integrations Partner
+  description: "Build, discover and connect with third party integration applications published to the Dwolla Integration Directory."
+---
+
+import Button from "../../components/base/Button";
+import CardGrid from "../../components/base/CardGrid";
+import Card from "../../components/base/Card";
+import plaid from "../../assets/images/content-images/content-icons/partner-logos/plaid-logo-simple.svg";
+import sift from "../../assets/images/content-images/content-icons/partner-logos/sift-logo-simple.svg";
+import authorization from "../../assets/images/content-images/content-icons/integration-authorization.svg";
+
+# Partner Integrations
+
+Dwolla is partnering with industry leaders in key areas to build out a network of service providers that ultimately help Dwolla clients expand their business.
+
+As an integration partner, you can build a third-party integration app to capture authorization from a Dwolla client which gives you permission to interact with data stored in Dwolla. The capabilities of custom integrations are endless and can include applications that send transaction events as Slack messages or import transaction data into Quickbooks.
+
+The Dwolla Partner Ecosystem, as a whole, provides our API Clients with seamless opportunities to provide a more complete offering for their platform and Ecosystem Partners with easier access to a new segment of potential customers. With a technical integration, you open up additional opportunities for growth and usage for your app.
+
+## Applying to the Integration Partner Program
+
+Before creating an integration application, you must apply to become an integration partner. Our partnerships team will guide you through the process of developing and publishing an integration to Dwolla’s Integration Directory. To ensure a seamless and secure integration, all integration partners will follow a set of guidelines and requirements prior to becoming listed as an approved partner.
+
+<Button
+  text="Apply to be an Integration Partner"
+  size="large"
+  variant="primary"
+  link={{
+    href: "https://www.dwolla.com/integrations/#cta-section",
```

Comment:
I think this may have changed to https://www.dwolla.com/integrations/#cta-form-0

Reply:
Good catch, Stephen! Should be fixed with my latest commit